### PR TITLE
CDAP-7628 system artifact loader thread logging

### PR DESCRIPTION
**Pull request by gsps1**
Add error logging in system artifact loader.

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java [hunk 1]
```diff
@@ -42,10 +48,12 @@ public Service get() {
           protected void doStart() {
             try {
               artifactRepository.addSystemArtifacts();
+              LOG.info("Successfully added system artifacts");
               // if there is no exception, all good, continue on
               notifyStarted();
             } catch (Exception e) {
               // transient error, fail it and retry
+              ERROR_LOG.warn("Exception while adding system artifacts", e);
```
Comment (gokulavasan) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java]: The JIRA talks about limiting the retries to a specific number. I see it is not part of this PR.

Comment (gsps1) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java]: we can wrap with another retry strategy with a limit on number of retries. but right now i am not sure if we should give up after `n retries`

Comment (albertshau) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java]: I don't think we need to have a cap on retries.

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java [hunk 2]
```diff
@@ -42,10 +48,12 @@ public Service get() {
           protected void doStart() {
             try {
               artifactRepository.addSystemArtifacts();
+              LOG.info("Successfully added system artifacts");
```
Comment (albertshau) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java]: this can be a misleading message as this can complete without adding anything.  It already logs when a system artifact is added and when an artifact is skipped, don't think we need an additional message.

Comment (gsps1) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java]: removed this log message.

**Comment by gsps1** [comment]
@albertshau addressed comment, please review again. thanks

**Review by albertshau — APPROVED** [review]
lgtm
